### PR TITLE
Feature/conditional rendering

### LINKED ISSUE #7: Render individual movie details
As a user, when I click on the a movie poster from the main page, I want to be taken to another page that shows me more details about the movie.

### LINKED ISSUE #8: Hide home screen on individual movie view
As a user, when I'm on a page for a specific movie, I don't want to be distracted by all of the other content from the main page.

### LINKED ISSUE #9: Navigation functionality
As a user, I want to be able to freely move around the app, no matter what I decide to click on. There should always be a way to return to the home page.

### PR DESCRIPTION
**Pull request by baileydunning**
### What does this PR do?
+ Add conditional rendering to App component that determines what is shown on the UI based on whether or not the user has selected a movie.
+ Add functionality to header/title that returns user to main page
+ Add initial CSS styling to clean up UI

### What files were changed?
+ `App.js`
+ `Movie.js`
+ `src/css`

### Next Steps:
+ Pseudocode tests 

### Issues:
+ Closes #7, Closes #8, Closes #9 

### Screenshots
Screen Shot 2020-12-03 at 8 18 45 AM
Screen Shot 2020-12-03 at 8 19 02 AM

### CODE REVIEW

**Comment by baileydunning** [comment]
Hey @bruce-gordon, just a thought: it might be beneficial to reorganize our file structure and make folders for each component (containing the jsx, test and css) on our next branch

File: public/index.html [hunk 1]
```diff
@@ -24,20 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Rancid Tomatillos</title>
```
Comment (bruce-gordon) [inline public/index.html:27]: Good call.

File: src/App.js [hunk 2]
```diff
@@ -9,15 +10,40 @@ class App extends Component {
     super()
     this.state = {
       movies: movieData,
-      isMain: true
+      selectedMovie: null
```
Comment (bruce-gordon) [inline src/App.js:13]: I feel really good about the logic we came up with here.

File: src/components/Movie.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+const Movie = ({ movie }) => {
+  return (
+    <section className='movie-section' style={{ background: `url(${ movie.backdrop_path })`}}>
+      <article className="movie-info">
+        <img src={ movie.poster_path } alt="movie-poster" className="movie-poster"></img>
+        <div className="movie-text">
+          <h2>{movie.title}</h2>
+          <p><b>Average Rating:</b> {movie.average_rating.toFixed(1)}</p>
+          <p><b>Release Date:</b> {new Date(movie.release_date).toDateString()}</p>
```
Comment (bruce-gordon) [inline src/components/Movie.js:12]: Good addition for displaying the date the way we want it.  Nice job.

**Review by bruce-gordon — APPROVED** [review]

**Pull request merged**